courseware/data_literacy/about_the_course.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: About the Course
+nav_order: 1
+parent: Data Literacy with Data Commons
+grand_parent: Courseware
+---
+
+# About the Course
+
+<div markdown="span" class="alert alert-info" role="alert" style="text-align: center">
+    >> [Access the course materials here](course_materials.html). <<
+</div>
+
+## What is it?
+
+A set of [modules](course_materials.html#modules) focusing on several [key concepts](topics.html#key-concepts) focusing on data modeling, analysis, visualization and the (ab)use of data to tell (false) narratives. Each module lists its objectives and builds on a pedagogical narrative around the explanation of key concepts, e.g. the differences between correlations and causation. We extensively use the Data Commons platform to point to _real world_ examples without needing to write a single line of code!  
```

Comment:
e.g. -> e.g.,

Reply:
Done! Learned this is a British vs American English difference.